testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/messaging/AddAcceptorOnlineTest.java

```diff
@@ -0,0 +1,94 @@
+package org.wildfly.extras.creaper.commands.messaging;
+
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.ManagementClient;
+import org.wildfly.extras.creaper.core.online.ModelNodeResult;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.OnlineOptions;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+
+import java.io.IOException;
+
+@RunWith(Arquillian.class)
+public class AddAcceptorOnlineTest {
+
+    private static final String TEST_ACCEPTOR = "testAcceptor";
+
+    private OnlineManagementClient client;
+    private Operations ops;
+
+    @Before
+    public void connect() throws IOException {
+        client = ManagementClient.online(OnlineOptions.standalone().localDefault().build());
+        ops = new Operations(client);
+    }
+
+    @After
+    public void after() throws CommandFailedException, IOException, OperationException {
+        ops.removeIfExists(
+                MessagingUtils.address(client, MessagingUtils.DEFAULT_SERVER_NAME)
+                        .and("remote-acceptor", TEST_ACCEPTOR));
+        ops.removeIfExists(
+                MessagingUtils.address(client, MessagingUtils.DEFAULT_SERVER_NAME)
+                        .and("acceptor", TEST_ACCEPTOR));
+        ops.removeIfExists(
+                MessagingUtils.address(client, MessagingUtils.DEFAULT_SERVER_NAME)
+                        .and("in-vm-acceptor", TEST_ACCEPTOR));
+        client.close();
+    }
+
+    @Test
+    public void addInVmAcceptor() throws CommandFailedException, IOException {
+        int serverId = 145872154;
+
+        AddAcceptor addAcceptor = new AddAcceptor
+                .InVmBuilder(TEST_ACCEPTOR)
```

Comment:
I can sympathise with what you're trying to do, but this is definitely not idiomatic. Keep the entire FQN of the class on a single line: `new AddAcceptor.InVmBuilder(TEST_ACCEPTOR)`.